src/Components/TableRow.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+
+function TableRow(props) {
+  const [selectedRow, setSelectedRow] = useState(null);
+  const setRowStatus = () => {
+    if (selectedRow === null) {
+      setSelectedRow("table-active");
+    } else {
+      setSelectedRow(null);
```

Comment:
That's a clever solution🙂 However I would suggest slightly different approach:
1. The value in state could be a boolean called `isSelected` initially set to `false`.
3. The `onClick` handler could be a function that changes `isSelected` to the opposite value when clicked:
```
setIsSelected(!isSelected)
```
`!` operator will basically change `true` to `false` and `false` to `true`.
4. Then you could add a ternary operator when defining a `className` attribute, that assigns a class name depending on the value of `isSelected`. For example if it's true - assign the class name with highlighted background - otherwise assign a different one or even null:
```
className={isSelected ? "table-active" : null}
```

Reply:
I started off with this approach actually but got stuck then came up with this solution. I got stuck on where should I have my ternary statement, but now that you handed it to me I'll re-werite it.

Reply:
Done